src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorModificationInfos.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.server.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.ReporterModel;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.NetworkModificationException;
+import org.gridsuite.modification.server.entities.equipment.modification.ShuntCompensatorModificationEntity;
+import org.gridsuite.modification.server.modifications.AbstractModification;
+import org.gridsuite.modification.server.modifications.ShuntCompensatorModification;
+
+/**
+ * @author Seddik Yengui <Seddik.yengui at rte-france.com>
+ */
+
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString(callSuper = true)
+@Schema(description = "Shunt compensator modification")
+public class ShuntCompensatorModificationInfos extends BasicEquipmentModificationInfos {
+
+    @Schema(description = "voltage level id")
+    private String voltageLevelId;
+
+    @Schema(description = "Susceptance per section")
+    private AttributeModification<Double> susceptancePerSection;
+
+    @JsonProperty("qAtNominalV")
```

Comment:
Why do you need this @JsonProperty annotation here ?

Reply:
Because the field began with lowercase followed by uppercase, the generated getter and setter are : setqAtNominalV and getqAtNominalV instead of setQAtNominalV and getQAtNominalV.therefore, the value of qAtNominalV will not be affected during deserialization. @JsonProperty indicate which prop to look for in th json

Reply:
ok